migration_steps/initialise_environments/delete_existing_data/app.py

```diff
@@ -70,13 +91,28 @@ def main():
                     sql_statement = ""
     log.info(f"Finished preparing delete statements to run")
 
-    if os.environ.get("ENVIRONMENT") in ["local", "preproduction", "preqa", "qa"]:
-        for statement in sql_statements:
-            log.info(f"Running statement: \n{statement}")
-            response = target_db_engine.execute(statement)
-            log.info(f"{response.rowcount} rows updated\n")
-    else:
-        log.info(f"Not running delete statements {os.environ.get('ENVIRONMENT')}!")
+    for statement in sql_statements:
+        log.info(f"Running statement: \n{statement}")
+        response = target_db_engine.execute(statement)
+        log.info(f"{response.rowcount} rows updated\n")
+
+    log.info("Checking deletion table counts")
+    stop_migration = False
+    for table in sirius_deletion_check_tables:
+        statement = f"SELECT id FROM deletions.deletions_{table};"
+        response = target_db_engine.execute(statement)
+        row_count = response.rowcount
+        log.info(f"Found {row_count} records in deletions.deletions_{table}. Expected 0.")
+        if row_count == 0:
```

Comment:
My personal preference would be to do something like:
```
if row_count > 0:
  ...stop the migration here...
```
I find the `continue` syntax a bit misleading in this context.

Reply:
Agreed, it's not needed here. I've got a habit of exiting ifs early to keep a flatter code structure - some bad previous experience of massive if statements nested several levels deep :D